README.md

```diff
@@ -1,209 +1,350 @@
-# Dart wrapper library for [facebook/react](http://facebook.github.io/)
+# Dart wrapper for [React JS](https://reactjs.org/)
 
 [![Pub](https://img.shields.io/pub/v/react.svg)](https://pub.dartlang.org/packages/react)
-[![documentation](https://img.shields.io/badge/Documentation-react-blue.svg)](https://www.dartdocs.org/documentation/react/latest/)
+![ReactJS v15.6.0](https://img.shields.io/badge/React_JS-v15.6.0-green.svg)
+[![Build Status](https://travis-ci.com/cleandart/react-dart.svg?branch=master)](https://travis-ci.com/cleandart/react-dart)
+[![React Dart API Docs](https://img.shields.io/badge/api_docs-react-blue.svg)](https://pub.dartlang.org/documentation/react/latest/)
 
-## Getting started
+## Getting Started
+
+### Installation
 
 If you are not familiar with the ReactJS library, read this [react tutorial](http://facebook.github.io/react/docs/getting-started.html) first.
 
-To integrate in Dart project add dependency [react](https://pub.dartlang.org/packages/react) to pubspec.yaml.
+1. Install the Dart SDK
+
+    ```bash
+    brew install dart
+    ```
+    
+2. Create a `pubspec.yaml` file in the root of your project, and add `react` as a dependency: 
+
+    ```yaml
+    name: your_package_name
+    version: 1.0.0
+    environment:
+      sdk: ^2.0.0 
+    dependencies:
+      react: ^4.5.0
+    ```
+
+3. Install the dependencies using pub:
+
+    ```bash
+    pub get
+    ```
+
+### Wire things up
+
+#### HTML
 
-Include the native javascript `react` and `react_dom` libraries (provided with this library for compatibility reasons) in `index.html` and create element where you'll mount the react component you'll create.
+In a `.html` file where Include the native javascript `react` and `react_dom` libraries 
+_(provided with this library for compatibility reasons)_ within your `.html` file, 
```

Comment:
This sentence reads weird, I think maybe "where" should be "we"? and it's redundant because it starts with "In a `.html` file" and ends with "within your `.html` file"

Reply:
Will address in a follow-up PR